dev-docs/bidders/ownadx.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: bidder
+title: OwnAdX
+description: Prebid OwnAdX Bidder Adaptor
+biddercode: ownadx
+gdpr_supported: false
+usp_supported: false
+coppa_supported: false
+gpp_supported: false
+schain_supported: false
+dchain_supported: false
+userId:
+media_types: banner, video
+safeframes_ok: false
+deals_supported: false
+floors_supported: true
+fpd_supported: false
+pbjs: false
+pbs: true
+pbs_app_supported: true
+prebid_member: false
+multiformat_supported: will-bid-on-one
+ortb_blocking_supported: false
+---
+
+### Note:
+
+The Example Bidding adapter requires setup before beginning. Please contact us at support@techbravo.com
+
+### Bid Params
+
+{: .table .table-bordered .table-striped }
+| Name          | Scope    | Description           | Example                  | Type      |
+|---------------|----------|-----------------------|--------------------------|-----------|
+| `host`        | required | Host subdomain             | `test`                   | `string`  |
```

Comment:
All of these string params needs quotes in the example column. The backtick is for highlighting and won't show up in the final display.